.travis.yml

```diff
@@ -22,11 +24,35 @@ matrix:
         - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
         - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
         - SPARK_VERSION=2.4.5
-        - PANDAS_VERSION=0.24.2
+        - PANDAS_VERSION=1.0.1
         - PYARROW_VERSION=0.13.0
         - KOALAS_USAGE_LOGGER='databricks.koalas.usage_logging.usage_logger'
     # When we upgrade Spark 3.0 for this, we could try to just use JDK 11 at 'xenial'.
     # Python 3.7 and higher are only available at this image.
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      jdk:
+        - oraclejdk8
+      env:
+        - PATH=$(echo "$PATH" | sed -e 's/:\/usr\/local\/lib\/jvm\/openjdk11\/bin//')
+        - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+        - SPARK_VERSION=2.4.5
+        - PANDAS_VERSION=1.0.1
+        - PYARROW_VERSION=0.14.1
+
+    # For testing, we temporarily test pandas<1.0.0 and pandas>=1.0.0 both for python3.6 and 3.7.
+    # TODO: So, we should remove below tests before PR #1197 is merged.
```

Comment:
@itholic can you file separate issue for each todo?

Reply:
yes i will. thanks for the review :)